src/target/move.c

```diff
@@ -334,12 +322,16 @@ static void move_side(enum step_direction side)
 {
 	enable_walls_control();
 	target_straight(current_cell_start_micrometers,
-			0.02 - MOUSE_AXIS_SEPARATION / 2., 0.448);
+			CELL_DIMENSION / 2 - get_turn_radius() -
+			    MOUSE_AXIS_SEPARATION / 2.,
+			get_turn_linear_speed());
 	disable_walls_control();
 	turn_side(side);
 	enable_walls_control();
 	target_straight(get_encoder_average_micrometers(),
-			0.02 + MOUSE_AXIS_SEPARATION / 2., max_linear_speed);
+			-CELL_DIMENSION / 2 + get_turn_radius() +
```

Comment:
Why changing the sign here? I think it should be `CELL_DIMENSION / 2 - get_turn_radius()`, just like in the `target_straight` call above.

Reply:
ok